### DISCUSSION TITLE
Still have confused on Configure SSL with Step by step installation from Documentation.

### DISCUSSION THREAD

**Discussion by DitaAjiPratama** [searx/searx#3216] [Installation issue: Official wiki] [answered]
I install searx on my server with this instruction:
https://searx.github.io/searx/admin/installation-searx.html#installation-basic
I use default setting and change bind_address into my domain name. It works.
Now I still wonder how to configure SSL on my searx installation. Is there someone who can help me?

**Comment by virtadpt** [comment, marked as answer]
There are a couple of pages in the Searx wiki that describe how to go about that.  Basically, you put a web server in front of Searx to proxy requests, cache results, and implement HTTPS.

* https://searx.github.io/searx/admin/installation-nginx.html
* https://searx.github.io/searx/admin/installation-apache.html

**Reply by DitaAjiPratama** [reply]
Did I need reinstall my searx again with filtron or morty?

**Reply by DitaAjiPratama** [reply]
Sorry my fault, Installing using nginx is the next step for installation, not another new fresh installation method. 
I'm getting clear now after seeing this diagram:
https://searx.github.io/searx/admin/architecture.html#arch-public
That mean I can create ssl in nginx and using proxy for searx.
Thank you for helping.
